imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,82 @@
+from django.test import TestCase
+import factory
+from . import models
+from .models import Photos, Album
+from django.contrib.auth.models import User
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = models.User
+        django_get_or_create = ('username',)
```

Comment:
be aware, the meaning of this setting is that the factory will look at the username attribute of a newly-requested user and will create a new user only if there is not already one in place.